deploy/common/k8sbased.sh

```diff
@@ -163,6 +163,25 @@ function launch_central {
     elif [[ -x "$(command -v roxctl)" && "$(roxctl version)" == "$MAIN_IMAGE_TAG" ]]; then
       echo "Using $(command -v roxctl) for install due to version match with MAIN_IMAGE_TAG $MAIN_IMAGE_TAG"
       use_docker=0
+    else
```

Comment:
I'd skip this in CI. This script is pretty fragile and changes to it tend to break CI.
```suggestion
    elif [[ -z "${CI}" ]]; then
```